crates/sui-indexer-alt-framework/src/pipeline/concurrent/mod.rs

```diff
@@ -417,7 +413,8 @@ mod tests {
                     .with_epoch(1)
                     .with_network_total_transactions(checkpoint * 2)
                     .with_timestamp_ms(1000000000 + checkpoint * 1000)
-                    .build_checkpoint(),
+                    .build_checkpoint()
+                    .into(),
```

Comment:
Similar to the comment in local_client.rs, it might be worth adding a `TestCheckpointBuilder` as well because I figure we'd want to generate checkpoints with the newly added fields for testing. And then we won't have to `into` here and in other places.